airlock/nav.py

```diff
@@ -12,7 +13,7 @@ def default_predicate(request):
 class NavItem:
     name: str
     url_name: str
-    predicate: Callable = default_predicate
+    predicate: Callable[[HttpRequest], bool] = default_predicate
```

Comment:
An interesting type signature -- but looks like `predicate` is never not the default.  @bloodearnest can we drop it?

Reply:
I imported this nav code from job-server, so not 100% sure.